### DISCUSSION TITLE
Native Window configurations don't live reload

### DISCUSSION THREAD

**Discussion by Slanderkin** [zauberzeug/nicegui#892] [General]
I noticed in my own testing that when you change the minimium size of the window with a live reload that it doesn't actually change this window configuration option. I believe (as reasoned below) this will affect basically all window configuration parameters when live reloading.

Test it here:
```Python
from nicegui import ui,app

app.native.window_args['resizable'] = True
#app.native.window_args['min_size'] = (400,400) #After initial run, uncomment me then reload
app.native.window_args['min_size'] = (1000,1000) #After initial run, comment me

ui.label('Hello world')

ui.run(title='Test', dark=True, native=True)
```

I don't think this is an issue with NiceGui as looking at pywebview this config option is set on window create so it won't take affect when a new window is not created (like in the live reload). I guess that window start will have the same thing happen although I didn't test that.

My main goal with this post is to just raise awareness that this is the case if someone in the future runs across this thinking its a NiceGui bug.

**Comment by phoskee** [comment]
[See this](https://github.com/zauberzeug/nicegui/discussions/770#discussioncomment-5636320)

**Comment by falkoschindler** [comment]
Thanks for pointing this out, @Slanderkin!
Indeed, some `ui.run` parameters don't take effect before restarting the main process. This includes window arguments as well as server arguments like `host` and `port`.
